config/routes.rb

```diff
@@ -1,6 +1,30 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :tweets do
+    member do
+      get 'stats', to: 'tweets#tweet_stats'
+      post 'create', to: 'tweets#create'
+      put 'update', to: 'tweets#update'
+      post 'reply', to: 'replies#create'
+      post 'quote', to: 'quotes#create'
+      post 'bookmark', to: 'bookmarks#create'
+      post 'retweet', to: 'retweets#create'
+      post 'like', to: 'likes#create'
+      delete 'unlike', to: 'likes#destroy'
+    end
+  end
```

Comment:
You could have done the same here without using member only by using resources like this

```suggestion
  resources :tweet, only: [:create, :update] do
    member do
      resources :bookmark, only: [:create]
      resources :reply, only: [:create]
      resources :quote, only: [:create]
      resources :retweet, only: [:create]
      get 'stats', to: 'tweets#tweet_stats'
      post 'like', to: 'likes#create'
      delete 'unlike', to: 'likes#destroy'
    end
  end
```